src/esp/physics/bullet/objectWrappers/ManagedBulletArticulatedObject.h

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) Facebook, Inc. and its affiliates.
+// This source code is licensed under the MIT license found in the
+// LICENSE file in the root directory of this source tree.
+
+#ifndef ESP_PHYSICS_MANAGEDBULLETARTICULATEDOBJECT_H_
+#define ESP_PHYSICS_MANAGEDBULLETARTICULATEDOBJECT_H_
+
+#ifdef ESP_BUILD_WITH_BULLET
+#include "esp/physics/bullet/BulletArticulatedObject.h"
+#endif
+#include "esp/physics/objectWrappers/ManagedArticulatedObject.h"
+
+namespace esp {
+namespace physics {
+
+/**
+ * @brief Class describing wrapper for dynamic ArticulatedObject constructions
+ * using the Bullet library. Provides bindings for all
+ * ArticulatedObject-specific functionality.
+ */
+class ManagedBulletArticulatedObject
+    : public esp::physics::ManagedArticulatedObject {
+ public:
+  ManagedBulletArticulatedObject()
+      : ManagedArticulatedObject("ManagedBulletArticulatedObject") {}
+
+#ifdef ESP_BUILD_WITH_BULLET
+  bool contactTest() {
+    if (auto sp = getBulletObjectReference()) {
+      return sp->contactTest();
+    }
+    return false;
+  }
+
+ protected:
+  /**
+   * @brief Templated version of obj ref getter. This function accesses the
+   * underlying shared pointer of this object's @p weakObjRef_ if it exists,
+   * and casts it to the specified template parameter; if it the ptr does not
+   * exist, it provides a message.
+   * @return Either a shared pointer of this wrapper's object, or nullptr if
+   * dne.
```

Comment:
This comment could be clearer - I was planning on a different format when I wrote it, and didn't get back to it before it got merged.  Mea culpa :).

Reply:
Seems fine to me. However, you can propose a change if you'd like.

Reply:
Just have it specify that it is recasting explicitly to a BulletArticulatedObject, and not to some user-accessible template param.